hot-reload.md

```diff
@@ -7,7 +7,7 @@ permalink: /hot-reload/
 * TOC
 {:toc}
 
-## Using hot reload
+## Using hot reload to quickly view changes
```

Comment:
Prefer the previous version. Headings must be simple and short. The splash page on flutter.io says this, so users will already know. Also, we elaborate below.

Reply:
Change reverted